src/parsers/pluscal.ts

```diff
@@ -91,11 +94,49 @@ export class TranspilerStdoutParser extends ProcessOutputHandler<DCollection> {
         return true;
     }
 
+    
     private addError(location: vscode.Position, message: string) {
         const locRange = new vscode.Range(location, location);
         this.result.addMessage(this.filePath, locRange, message);
     }
 
+    /**
+     * Adds info on labels added by the pluscal translated.
+     * 
+     * Only takes effect if the `-reportLabels` was added to PlusCal options. Output looks like:
+     * 
+     *       The following labels were added: // 1
+     *           Lbl_1 at line 16, column 5
+     *           Lbl_2 at line 23, column 9
+     * 
+     * 
+     * So we can start looking for labels as soon as we see (1)
+     * and stop as soon as we stop seeing label strings.
+     */
+    private tryParseAddedLabels(line: string) {
+        if (line.startsWith("The following label")) { // could be `was added` or `were added`
+            this.nowParsingAddedLabels = true;
+            return true;
+        };
+        if (!this.nowParsingAddedLabels) {
+            return false
+        }
+
+        const matcher = /^\s\s([A-Za-z0-9_]+) at line (\d+), column (\d+)$/g.exec(line)
+        if (!matcher) { // done parsing 
+            this.nowParsingAddedLabels = false;
+            return false;
+        }
+
+        const labelname = matcher[1];
+        const message = `Missing label, translator inserted \`${labelname}\` here`;
```

Comment:
https://github.com/tlaplus/tlaplus/blob/21f924cc0d77501ae2de104dda4f56997a29bc5b/tlatools/org.lamport.tlatools/src/pcal/ParseAlgorithm.java#L654-L666

Reply:
No, lets just add a code comment as a pointer to where Pcal emits this string.